docs/content/en/docs/how-tos/buildpacks/_index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Building Artifacts with CNCF Buildpacks"
+linkTitle: "Buildpacks"
+weight: 100
+---
+
+This page describes building Skaffold artifacts with [buildpacks](https://buildpacks.io/).
+Buildpacks enable building language-based containers from source code, without the need for a Dockerfile.
+
+To use buildpacks with Skaffold, please install the following tools:
+
+* [pack](https://buildpacks.io/docs/install-pack/)
+* [docker](https://docs.docker.com/install/)
+
+## Tutorial
+
+Clone the Skaffold buildpacks [example](https://github.com/GoogleContainerTools/Skaffold/blob/master/examples/buildpacks/) for sample code.
+
+Set the default buildpack to one that can build Go applications: 
+
+``` shell
+$ pack set-default-builder heroku/buildpacks
+```
+
+Now, you should be able to use Skaffold:
+
+```shell
+$ skaffold run --tail --default-repo <your repo>
+```
+This will deploy Hello World in Go to your cluster.
```

Comment:
Do we have cluster step up steps somewhere that we can point users to if they have never set up a cluster `kubectl `?

Reply:
I think the getting-started guide might have it. If not we shd add it there.
Maybe we can do that in a separate PR.